tests/api_fastapi/core_api/routes/public/test_dag_run.py

```diff
@@ -432,6 +469,333 @@ def test_invalid_state(self, test_client):
         )
 
 
+class TestListDagRunsBatch:
+    @staticmethod
+    def parse_datetime(datetime_str):
+        return datetime_str.isoformat().replace("+00:00", "Z") if datetime_str else None
+
+    @staticmethod
+    def get_dag_run_dict(run: DagRun):
+        return {
+            "run_id": run.run_id,
```

Comment:
Once https://github.com/apache/airflow/pull/44305 is merged, I need to take changes and update tests

Reply:
Done